### PR TITLE
Related links

### PR DESCRIPTION
**Pull request by gielfeldt**

### CODE REVIEW

File: kkb_group/kkb_group.features.features_overrides.inc [hunk 1]
```diff
@@ -12,8 +12,8 @@ function kkb_group_features_override_default_overrides() {
   $overrides = array();
 
   // Exported overrides for: field_instance
+  $overrides["field_instance.node-ding_group-field_ding_group_body.required"] = 0;
   $overrides["field_instance.node-ding_group-field_ding_group_list_image.required"] = 1;
-  $overrides["field_instance.node-ding_group-field_ding_group_title_image.required"] = 1;
```
Comment (kasperg) [inline kkb_group/kkb_group.features.features_overrides.inc]: Are these changes intentional?

File: kkb_group/kkb_group.features.inc [hunk 2]
```diff
@@ -17,10 +17,10 @@ function kkb_group_ctools_plugin_api($module = NULL, $api = NULL) {
...
 function kkb_group_field_default_field_instances_alter(&$data) {
+  if (isset($data['node-ding_group-field_ding_group_body'])) {
+    $data['node-ding_group-field_ding_group_body']['required'] = 0; /* WAS: 1 */
+  }
   if (isset($data['node-ding_group-field_ding_group_list_image'])) {
     $data['node-ding_group-field_ding_group_list_image']['required'] = 1; /* WAS: 0 */
   }
-  if (isset($data['node-ding_group-field_ding_group_title_image'])) {
-    $data['node-ding_group-field_ding_group_title_image']['required'] = 1; /* WAS: 0 */
-  }
```
Comment (kasperg) [inline kkb_group/kkb_group.features.inc]: Same question as above.

File: kkb_group/kkb_group.info [hunk 3]
```diff
@@ -8,9 +8,9 @@ dependencies[] = field_group
 dependencies[] = file
 features[ctools][] = field_group:field_group:1
 features[features_api][] = api:2
+features[features_overrides][] = field_instance.node-ding_group-field_ding_group_body.required
 features[features_overrides][] = field_instance.node-ding_group-field_ding_group_list_image.required
-features[features_overrides][] = field_instance.node-ding_group-field_ding_group_title_image.required
```
Comment (kasperg) [inline kkb_group/kkb_group.info]: And again :).

File: kkb_related_links/kkb_related_links.features.field_instance.inc [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+      'teaser_highlight' => array(
+        'label' => 'above',
+        'settings' => array(),
+        'type' => 'hidden',
+        'weight' => 0,
+      ),
+    ),
+    'entity_type' => 'node',
+    'field_name' => 'field_related_links',
+    'label' => 'Relaterede links',
```
Comment (kasperg) [inline kkb_related_links/kkb_related_links.features.field_instance.inc]: This should be kept in english for them to translate?

File: kkb_related_links/kkb_related_links.features.field_instance.inc [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+      'weight' => 31,
+    ),
+  );
+
+  // Exported field_instance: 'node-ding_page-field_related_links'
+  $field_instances['node-ding_page-field_related_links'] = array(
+    'bundle' => 'ding_page',
+    'default_value' => NULL,
+    'deleted' => 0,
+    'description' => '',
```
Comment (kasperg) [inline kkb_related_links/kkb_related_links.features.field_instance.inc]: How about adding a description to help editors along a bit?

**Comment by kasperg** [comment]
Review complete. I asume that no changes are required to display the field as intended?

File: kkb_group/kkb_group.features.features_overrides.inc (hunk 1 above)
Comment (gielfeldt) [inline kkb_group/kkb_group.features.features_overrides.inc]: Yes. Title image should not be required anymore. The "required" state was a kkb override.

File: kkb_group/kkb_group.features.inc (hunk 2 above)
Comment (gielfeldt) [inline kkb_group/kkb_group.features.inc]: Same answer as above.

File: kkb_group/kkb_group.info (hunk 3 above)
Comment (gielfeldt) [inline kkb_group/kkb_group.info]: And again :).

File: kkb_related_links/kkb_related_links.features.field_instance.inc (hunk 4 above)
Comment (gielfeldt) [inline kkb_related_links/kkb_related_links.features.field_instance.inc]: Hmmm. Not sure. Let me check.

File: kkb_related_links/kkb_related_links.features.field_instance.inc (hunk 5 above)
Comment (gielfeldt) [inline kkb_related_links/kkb_related_links.features.field_instance.inc]: Of course.

**Comment by gielfeldt** [comment]
Actually ... it looks like I might have missed the export of the display settings. Let me check.

File: kkb_related_links/kkb_related_links.features.field_instance.inc (hunk 4 above)
Comment (gielfeldt) [inline kkb_related_links/kkb_related_links.features.field_instance.inc]: I can't seem to get them translated? Maybe it's a field collection issue?

Comment (gielfeldt) [inline kkb_related_links/kkb_related_links.features.field_instance.inc]: Field Group can't be translated at all. Fields can be translated, but I can't seem to export the translations (because of translation context).

I suggest we default to English in the feature, and then translate manually afterwards?

@kasperg

File: kkb_group/kkb_group.features.features_overrides.inc (hunk 1 above)
Comment (kasperg) [inline kkb_group/kkb_group.features.features_overrides.inc]: Roger.
